examples/next/pages/ui/components/storage/storage-browser/custom-actions/styles.css

```
@@ -0,0 +1,4 @@
+// Intentionally to test error boundary. No effect in prod mode.
```

```suggestion
/* Intentionally to test error boundary. No effect in prod mode. */
```

Plain CSS only allows /* comments and not //